app/views/ppm_estimator/index.html.erb

```diff
@@ -0,0 +1,156 @@
+<%- @page_title = 'Personally Procured Move (PPM) Estimator' -%>
+
+<%= render partial: 'shared/contextual_header', locals: {page_title: @page_title, parent_pages: [{name: 'Tools & Resources', path: page_path('resources')}]} %>
+
+<div class="usa-section main-section">
+  <h1><%= @page_title %></h1>
+
+<h4>
+    For your upcoming move, get an estimate for how much the government will
+    pay <strong>you</strong> if you choose to "do it yourself."
+  </h4>
+
+  <p>
+    You can use this money to cover the cost of hiring packers and movers,
+    renting moving trucks, and so on. If you spend less to move than the
+    government will incentivize you to move yourself, you get to pocket the
+    difference!
+  </p>
+
+  <%= form_with url: ppm_estimator_path, class: 'ppm-form', id: 'ppm-estimate-form', method: 'get' do |form| %>
+    <fieldset>
+      <legend><div class="ppm-outline"></div>What is your rank?</legend>
+        <%= form.label :rank, 'Rank' %>
+        <%= form.select :rank, options_from_collection_for_select(@entitlements, :slug, :rank), { include_blank: true }, { id: :rank, required: true } %>
+    </fieldset>
+
+    <fieldset>
+      <legend><div class="ppm-outline"></div>What is your branch of service?</legend>
+        <%= form.label :branch, 'Branch of Service' %>
+        <%= form.select :branch, options_from_collection_for_select(@branches, :slug, :name), { include_blank: true }, { id: :branch, required: true } %>
+    </fieldset>
+
+    <fieldset>
+      <legend><div class="ppm-outline"></div>Do you have dependents?</legend>
+      <%= form.radio_button :dependents, 'yes', { id: :dependents_yes, required: true } %>
+      <%= form.label :dependents, raw('<strong>Yes</strong>, I have dependents'), :value => 'yes' %>
+      <%= form.radio_button :dependents, 'no', { id: :dependents_no, required: true } %>
+      <%= form.label :dependents, raw('<strong>No</strong>, I do not have dependents'), :value => 'no' %>
+    </fieldset>
+
+    <fieldset>
+      <legend><div class="ppm-outline"></div>Are you married?</legend>
+      <%= form.radio_button :married, 'yes', { id: :married_yes, required: true } %>
+      <%= form.label :married, raw('<strong>Yes</strong>, I am married'), :value => 'yes' %>
```

Comment:
A markup pattern you could consider here:

```erb
<%= form.label :married, value: 'yes' do %>
  <%= form.radio_button :married, 'yes', id: :married_yes, required: true %>
  <b>Yes</b>, I am married
<% end %>
```

This _should_ create HTML like:

```html
<label for="married">
  <input type="radio" id="married" required>
  <b>Yes</b>, I am married
</label>
```

☝️ I left out a couple of attributes in the rendered HTML, but you get the idea.

Reply:
It worked, although I had to put the form.radio_button tag above the form.label do tag.